Formula/git-remote-hg.rb

```diff
@@ -0,0 +1,17 @@
+class GitRemoteHg < Formula
+  desc "Transparent bidirectional bridge between Git and Mercurial"
+  homepage "https://github.com/felipec/git-remote-hg"
+  url "https://github.com/felipec/git-remote-hg/archive/v0.3.tar.gz"
+  sha256 "2dc889b641d72f5a73c4c7d5df3b8ea788e75a7ce80f5884a7a8d2e099287dce"
+
+  depends_on :hg
+  depends_on :python
```

Comment:
Tried this, but `git-remote-hg` needs `python2` binary so this `if` unfortunately breaks things as seen in http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/5779/ . What's the best approach in such case?